### PR TITLE
TCCP: Add "About us" sidebar, minor content updates

### PR DESCRIPTION
**Pull request by chosak**
This commit adds an "About us" sidebar to the TCCP landing page and card detail view.

## How to test this PR

After running a local server, visit the landing page at:

http://localhost:8000/consumer-tools/credit-cards/explore-cards/

Click "see all cards" then click on any card to see the detail page.

## Notes and todos

We have a few instances of this hardcoded text in the codebase, plus another instance of it as a Wagtail ReusableText snippet. It'd be nice if those were all unified somehow, but I'm leaving that for a future PR.

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

File: cfgov/tccp/jinja2/tccp/landing_page.html [hunk 1]
```diff
@@ -23,6 +23,18 @@ <h1>{{ heading }}</h1>
         </p>
     </div>
 
+    <div class="block block__sub">
+        <h2>
```
Comment (niqjohnson) [inline cfgov/tccp/jinja2/tccp/landing_page.html:27]:
```suggestion
        <h2 class="h3">
```

Looks like this is styled as an `h3` in the latest mockups.

File: cfgov/tccp/jinja2/tccp/landing_page.html [hunk 2]
```diff
@@ -40,9 +52,9 @@ <h1>{{ heading }}</h1>
             </div>
 
             <p>
-                If you don’t know your credit score,
-                <a href="/ask-cfpb/where-can-i-get-my-credit-scores-en-316/">find out for free</a>
-                or play around to find out how credit score affects rates.
+                If you don’t know it,
+                <a href="/ask-cfpb/where-can-i-get-my-credit-scores-en-316/"
+                    >find your credit score</a>.
             </p>
```
Comment (niqjohnson) [inline cfgov/tccp/jinja2/tccp/landing_page.html:58]: Looks like in the latest mockups this line is set as [helper text](https://cfpb.github.io/design-system/components/helper-text) and is ~ 5 px under the dropdown (so we can probably pull the `p` into the `div.o-form_group`

**Review by niqjohnson — APPROVED** [review]
Couple of small style tweaks, but those can wait until a separate PR if we want to merge this one sooner

File: cfgov/tccp/jinja2/tccp/landing_page.html (hunk 2 above)
Comment (chosak) [reply cfgov/tccp/jinja2/tccp/landing_page.html:58]: Great suggestion, just pushed a commit that does this!

**Pull request merged**